.github/workflows/nix_macos_apple_silicon.yml

```diff
@@ -39,11 +39,19 @@ jobs:
         run: cd examples/platform-switching/rust-platform && nix develop -c cargo test --release --locked
 
       - name: test launching the editor
-        run: cargo test --release --locked editor_launch_test::launch -- --ignored # `--ignored` to run this test that is ignored for "normal" runs
+        run: nix develop -c cargo test --release --locked editor_launch_test::launch -- --ignored # `--ignored` to run this test that is ignored for "normal" runs
 
       # we run the llvm wasm tests only on this machine because it is fast and wasm should be cross-target
       - name: execute llvm wasm tests with --release
         run: nix develop -c cargo test-gen-llvm-wasm --locked --release
 
-      - name: test website build script
-        run: REPL_DEBUG=1 bash www/build.sh
+      - name: set env var and test website build script
+        run: |
+          awk 'NR==1 {print; print "REPL_DEBUG=1"; next} 1' www/build.sh > tmpfile.txt && mv tmpfile.txt www/build.sh
```

Comment:
Interesting, what's this doing?

Also I'm surprised we're setting the debug flag in CI. The intention is that we shouldn't need that. I see it was here before but still curious.

Reply:
When using `nix develop -c` I can't just pass it the env var, so I used awk to write the env var into the script. I copied the env var setting from another workflow. I can try to see how it runs without.

Reply:
Oh I see, it's adding `REPL_DEBUG=1` as code to the script! Wow!
I see you tried lots of other ways of setting it so I guess.. fair enough!

This is disabling optimizations that we otherwise would do with the `wasm-opt` package.
And presumably that's because this is a test and we don't want to spend the time.
OK

Reply:
:+1: I'll check the runtime difference once it's done.

Reply:
There was no significant difference on macos or linux without the env var.

Reply:
OK great!
Nice work @Anton-4 😃